app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -94,6 +93,7 @@ import org.mozilla.fenix.perf.lazyMonitored
 import org.mozilla.fenix.settings.SupportUtils
 import org.mozilla.fenix.settings.advanced.getSelectedLocale
 import org.mozilla.fenix.share.SaveToPDFMiddleware
+import org.mozilla.fenix.share.SessionPrioritizationMiddleware
```

Comment:
I think we will need to add a new middleware to handle errors to the list, as this will replace the implementation that is coming from AC :)

Reply:
Hm, I believe this this new fenix middleware is what will handle the error. The one coming from AC `mozilla.components.browser.state.engine.middleware.SessionPrioritizationMiddleware` is removed the line above.

Reply:
I think we will need both? :) As we want to collect errors and have the functionality for tab prioritization from AC.

Reply:
Ahh I see, I moved the error handling into TelemetryMiddleware